versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/persist/ObjId.java

```diff
@@ -98,6 +98,14 @@ public static ObjId objIdFromString(@Nonnull @jakarta.annotation.Nonnull String
     }
   }
 
+  /**
+   * Creates an {@link ObjId} from its bytes representation, assuming that all remaining data in
```

Comment:
the `remaining` piece is a bit misleading... `ByteString` does not have a "pointer" (unlike `ByteBuffer`), does it?